README.md

```diff
@@ -44,4 +44,5 @@ Here are some web pages that you may need to refer to:
         it is not related to my major. Categorical fields would be Country/Region, and perhaps Province/State. 
         There are several numerical fields. Graphing the growth of cases over time for a selected state like 
         Michigan would be possible, but the date field looks different than what we have studied. I may need some help there.
-
+   - Hunter Vayda-Ramirez: I would like to use <https://github.com/awesomedata/awesome-public-datasets> by downloading the titantic.csv.zip file or   <https://github.com/GeostatsGuy/GeoDataSets/blob/master/dataset_101b.csv> (Not sure how to get).
```

Comment:
Both of these datasets look good. Please do some cleanup formatting here, though. Put each dataset as an indented bullet-point under your name. Then under each dataset say a little about what the data is about and whether it has numeric and categorical fields. I am mystified by the GeoDataSets/dataset_101b.csv -- is it earthquake data? Also mention why the dataset would be a good one, e.g., it relates to your major or minor or a societal issue, etc.

One way to download the dataset is to go up one level to GeoDataSets and press the download button on the right. Then choose "Download zip". You get all the datasets but you can just unzip the one dataset and then delete the zip. Since this is a small file, you can also just go to the dataset page on GitHub and choose the "Raw" button on the left. Then, on Windows, click Control-A to select all, and paste the whole dataset into a text editor.

Reply:
I think I fixed it okay now.